recipes/tensorflow-addons/build.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -ex
+
+export PATH="$PWD:$PATH"
+export CC=$(basename $CC)
+export CXX=$(basename $CXX)
+export LIBDIR=$PREFIX/lib
+export INCLUDEDIR=$PREFIX/include
+
+export TF_NEED_CUDA=0
+
+if [[ ${cuda_compiler_version} != "None" ]]; then
+    export TF_NEED_CUDA=1
+    export TF_CUDA_PATHS="${PREFIX},${CUDA_HOME}"
+    export TF_CUDA_VERSION="${cuda_compiler_version}"
+    export TF_CUDNN_VERSION="${cudnn}"
+    export GCC_HOST_COMPILER_PATH="${GCC}"
+    export CUDA_TOOLKIT_PATH="${CUDA_HOME}"
+    export CUDNN_INSTALL_PATH="${PREFIX}"
+
+    # addons uses a different nomenclature for compute capabilities than tf proper, see
+    # https://github.com/tensorflow/addons/blob/v0.14.0/build_deps/toolchains/gpu/cuda_configure.bzl#L18-L19
+    if [[ ${cuda_compiler_version} == 10.* ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5
+    elif [[ ${cuda_compiler_version} == 11.0* ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5,8.0
+    elif [[ ${cuda_compiler_version} == 11.1 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5,8.0,8.6
+    elif [[ ${cuda_compiler_version} == 11.2 ]]; then
+        export TF_CUDA_COMPUTE_CAPABILITIES=3.5,5.0,6.0,6.2,7.0,7.2,7.5,8.0,8.6
+    else
+        echo "unsupported cuda version."
+        exit 1
+    fi
+fi
+
+# This script links project with TensorFlow dependency; creates .bazelrc
+python ./configure.py
+# ironically, configure.py cannot be configured, so sed it is...
+sed -i'' -e "s:TF_HEADER_DIR=.*:TF_HEADER_DIR=\"$PREFIX/include\":" \
+    -e "s:TF_SHARED_LIBRARY_DIR=.*:TF_SHARED_LIBRARY_DIR=\"$PREFIX/lib\":" \
+    .bazelrc
+# show result (& sleep to allow log to catch up)
+cat .bazelrc
+sleep 2
+
+# use conda-forge wrapper for bazel, called bazel-toolchain
+bazel-toolchain build build_pip_pkg
```

Comment:
```suggestion
source gen-bazel-toolchain
bazel --crosstool_top=//bazel_toolchain:toolchain --target_cpu ${TARGET_CPU} build build_pip_pkg
```

Reply:
Thanks for the help on this! Unfortunately, this fails with:
```
Unknown startup option: '--crosstool_top=//bazel_toolchain:toolchain'.
```

Reply:
Thought I might still have to run `gen-bazel-toolchain` after `source`-ing it, but alas that didn't work either...

Reply:
You will need to put it behind the `build` in `bazel build`.

Reply:
> You will need to put it behind the `build` in `bazel build`.

It took me way too long, but now I think I got what you meant. 😅 

BTW, why are we adding `--target_cpu ${TARGET_CPU}`?

Reply:
> BTW, why are we adding `--target_cpu ${TARGET_CPU}`?

Apparently bazel is wondering the same thing, haha
```
ERROR: --target_cpu :: Unrecognized option: --target_cpu
```